optimism/src/mips/witness.rs

```diff
@@ -100,6 +100,100 @@ fn memory_size(total: usize) -> String {
 impl<Fp: Field> InterpreterEnv for Env<Fp> {
     type Variable = u32;
 
+    // Debug
+    fn debug_signed_16bits_variable(v: &Self::Variable) -> String {
+        // FIXME: It does suppose u32 is between 0 and 2 ^ 16
+        let v = *v;
+        // handle 0b1000000000000000
+        if v == 1 << 15 {
+            return String::from("0");
```

Comment:
Oh, so what's `0b0000000000000000`? Is it also zero, like are there two representations of zero?